server/controllers/requestController.js

```diff
@@ -0,0 +1,32 @@
+import requestsDb from '../dummyModel/request';
+/**
+ * class Request controls all request methods
+ * @class request
+ */
+export default class Requests {
+	/**
+	 * adds a new request
+	 * @param {object} req
+	 * @param {object} res
+	 * @returns {json} adds new request
+	 * @memberof Request
+	 */
+	static addRequest(req, res) {
+    const { title, department, equipment,serialNumber,description } = req.body;
+    const id = requestsDb.length;
```

Comment:
What is the essence of this line? The naming is not wonderful either.

Reply:
id is to give is rerquest an id, it checks the  current length of the request array and assigns the number as id

Reply:
it checks the size, but just so it can get the number as id, i do not want to hard code the id of each request, thats why.

Reply:
So where are you using this `id`?

Reply:
it will be gotten when a request is created, i will need it to update requests

Reply:
I don't see it being used anywhere else.

Reply:
it wasnt used. i will add it to json response.

Reply:
Okay. Give it the appropriate name. We would not sacrifice clarity for implementation

Reply:
okay, thank you